packages/loader/container-loader/src/containerContext.ts

```diff
@@ -90,15 +90,6 @@ export class ContainerContext implements IContainerContext {
 
     public readonly taggedLogger: ITelemetryLogger;
 
-    /**
-     * Subtlety: returns this.taggedLogger since vanilla this.logger is now deprecated. See IContainerContext for more
-     * details.
-    */
-    /** @deprecated See IContainerContext for more details. */
-    public get logger(): ITelemetryLogger {
```

Comment:
This change has been coming for so long I know it's fine, but makes me want to know - what is our promise of back compat for the combination of (old runtime, new loader)?  Is it n/n-2?

Reply:
@sdeshpande3 please double check, but I think our 0.55 runtime had: `context.taggedLogger  ?? new TaggedLoggerAdapter(context.logger);`

Reply:
0.45 is the current LTS version that we support:
https://github.com/microsoft/FluidFramework/blob/22239a50b175b7f8cb43689b0e947db2bbca6c5b/packages/test/test-version-utils/src/compatConfig.ts#L40

Reply:
0.45 loaders against older runtimes? The question was about new loaders, older runtimes. I think we only support loaders/drivers LTS as "^0.45.0". Or at least that is the matrix we test against.